src/modules/shared/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,98 @@
+@use "sass:map";
+@use "styles/abstracts" as *;
+
+.c-footer {
+	display: flex;
+	flex-flow: column;
+	padding: 0 $spacer-lg;
+	z-index: get-z-layer("base");
+	color: $neutral;
+
+	&__bar {
+		display: grid;
+		grid-template-columns: 1fr 2fr 1fr;
+		grid-template-rows: 1fr;
+		justify-items: center;
+		align-items: center;
+		padding-top: $spacer-md;
+		padding-bottom: $spacer-sm;
+		background-color: $white;
+		border-top: $divider-grey;
+
+		&--simple {
+			grid-template-columns: repeat(2, 1fr);
```

Comment:
Je kan uw 3 grid kolommen behouden en op de childs definiëren in welk grid kolom ze moeten staan dan is deze extra class ook niet nodig.